ADR/ADRXXX-architecture-decision-record-template.md

```diff
@@ -2,6 +2,10 @@
 
 Date: YYYY-MM-DD
 
+## Decision
+
+> what the team has decided to do
+
 ## Status
 
 > Proposed, Accepted, Rejected, Deprecated, Superseded by ADRzzz
```

Comment:
Personally I'd be tempted to put the status before the decision, otherwise I think it might get lost visually...

Reply:
I debated doing this too - given that we're allowing `Status: Rejected`, I think it probably is important that this doesn't get lost, so will move status back to the top